src/VirtualClient/VirtualClient.Main/profiles/PERF-IO-FIO-MULTITHROUGHPUT.json

```diff
@@ -12,7 +13,8 @@
         "TargetIOPS": "5000",
         "TargetPercents": "10,40,90,98,100,102,110",
         "DirectIO": true,
-        "InitializeDisksInParallel": true
+        "InitializeDisksInParallel": true,
+        "SequentialDiskCount": "1"
```

Comment:
SequentialDiskCount: Let's make the default behavior to leave this parameter as null/undefined. In the default scenario where the user does not define this, we apply random I/O to half of the disks and sequential I/O to the other half (e.g. if there are 4 disks, we apply random I/O to 2 and sequential I/O to 2). For cases where we have an odd number of disks, we should apply random I/O to just over half and sequential I/O to the rest (e.g. if there are 5 disks, we apply random I/O to 3 disks and sequential I/O to 2).

Reply:
I discussed this with Srivanthi
The Default case for OLTP is using 1 log disk(Sequential) & others as data disks(Random).